qcodes/instrument/channel.py

```diff
@@ -383,7 +383,9 @@ def lock(self) -> None:
         self._channels = tuple(self._channels)
         self._locked = True
 
-    def snapshot_base(self, update: bool=False, params_to_skip_update: Optional[Sequence[str]]=None):
+    def snapshot_base(self, update: bool = True,
+                      params_to_skip_update: Optional[Sequence[str]] = None
```

Comment:
this might be irrelevant for this PR, but i don't get why is `params_to_skip_update` NOT passed to the `chan.snapshot(update=update)` calls inside this method? it just seems to me that this feature is not well thought of and perhaps not well implemented. again, this is just nagging.